src/services/penaltyDocuments.js

```diff
@@ -269,12 +310,18 @@ export default class PenaltyDocument {
 			IndexName: 'ByOffset',
 			Limit: maxBatchSize,
 		};
+		let localOffset = offset;
+		const date = new Date();
+
+		if (typeof offset === 'undefined' || Number(offset) === 0) {
+			localOffset = subtractDays(date, this.daysToHold);
+		}
 
 		if (typeof offset !== 'undefined') {
```

Comment:
I don't think this condition/if statement is needed - within the block you're using the ```localOffset``` variable which will never be undefined at that point (given that the previous condition already gives it a value if it's falsy).